### ISSUE TITLE
MSID: 100788 Version: 1 DOI: 10.1101/2024.06.18.599531

### ISSUE THREAD

**Issue by fred-atherden** [elifesciences/publish-reviewed-preprints-issues#1496]
MSID: 100788

Version: 1

Preprint DOI: https://doi.org/10.1101/2024.06.18.599531

## Step 1. Awaiting reviews

**Editorial to post reviews via hypothesis**

**Useful links:**
- DocMap: https://data-hub-api.elifesciences.org/enhanced-preprints/docmaps/v2/by-publisher/elife/get-by-manuscript-id?manuscript_id=100788
- New model tracking: https://docs.google.com/spreadsheets/d/1_fHaoOy7hjyocptKtVJRijeNpUY4hBS7Ck_aVmx6ZJk/
- Reviews on sciety: https://sciety.org/articles/activity/10.1101/2024.06.18.599531

**For trouble shooting (e.g. no Docmaps available):**
- DocMap issue addressing: https://miro.com/app/board/uXjVNCwK6EI=/
- Explore DataHub DocMaps API: https://lookerstudio.google.com/reporting/4c2f0368-babb-4beb-b5b3-497e7e7b0f08/page/ejphD
- Unmatched submissions and preprints: https://lookerstudio.google.com/u/0/reporting/9f86204f-3bf7-477c-9b18-5c5ef141bf69/page/p_gxi57ha93c
- Unmatched manuscripts spreadsheet: https://docs.google.com/spreadsheets/d/15QcK8w-ssB7109RQEDtFpJPZ0J5HTGxoHa_2TtpMBbg/edit#gid=1336081641

## Step 2. Preview reviewed preprint

**Production QC content ahead of publication**

**Instructions:**
- QC preview: https://prod--epp.elifesciences.org/previews/100788v1
- Update ticket with any problems (add `blocked` label)
- When QC OK, add `QC OK` label to ticket and add publication date and time to https://docs.google.com/spreadsheets/d/1amAlKvdLcaDp5W8Z8g77NmkwbMF5n_u89ArSqPMO8jg
- Move card to next column
- (At end of the day post link in [#enhanced-preprint](https://elifesciences.slack.com/archives/C03EVJSUA77) and ask for PDF to be generated) 

**Useful links:**
- Preprint DOI :  https://doi.org/10.1101/2024.06.18.599531
- Confirm reviews returned by EPP: https://prod--epp.elifesciences.org/api/reviewed-preprints/100788/v1/reviews
- To update the MECA path in the docmap: https://docs.google.com/spreadsheets/d/1mctCQuNFBjSn97Lihy7_vBO6z7-N-oqyLv4clyi6zHg

## Step 3: Awaiting search reindex

**This step adds the reviewed preprint to the homepage: https://elifesciences.org**

The search reindex is triggered once an hour. We need the reviewed preprint to be indexed as the search application serves the journal homepage.

**Useful links:**
- Jenkins pipeline to reindex search can be triggered sooner or monitored here: https://alfred.elifesciences.org/job/process/job/process-reindex-reviewed-preprints/

## Step 4: Published! PDF requested

**Waiting for PDF to be generated**

**Useful links:**
- PDF tracking: https://docs.google.com/spreadsheets/d/106_XeDjmuBae7gexOTNzg60lapeqjl2aRn9DzupGyS8/

## Step 5: Introduce PDF to data folder and git repo

Upload PDF to relevent folder in git repo https://github.com/elifesciences/enhanced-preprints-data/

## Step 6: Done!

- [ ] Kettle is on!

**Comment by rkcook312** [comment]
@fred-atherden bit concerning the workflow for this has just rerun, not sure why?

**Comment by rkcook312** [comment]
Many thanks @fred-atherden!

**Comment by gnott** [comment]
Correcting my early statement on our call @fred-atherden, I was able to find the difference in the docmaps to account for why it was ingested again.

The docmap for `100788` at about 2024-11-19 10:30 UTC had extra steps, probably duplicate values for the existing steps. The higher peer review count would have caused it to run again. It might be possible for me to improve this, to avoid counting duplicate steps if they are present, however I think it was caused by something in data hub I haven't accounted for (yet).

Here's the JSON from that hour for reference (taken from `s3://elife-poa-packaging/docmaps/run_2024_11_19_0011/docmap_index/docmap_index.json`):

```json
{
    "@context": "https://w3id.org/docmaps/context.jsonld",
    "type": "docmap",
    "id": "https://data-hub-api.elifesciences.org/enhanced-preprints/docmaps/v2/by-publisher/elife/get-by-manuscript-id?manuscript_id=100788",
    "created": "2024-06-21T14:50:31+00:00",
    "updated": "2024-06-21T14:50:31+00:00",
    "publisher": {
        "account": {
            "id": "https://sciety.org/groups/elife",
            "service": "https://sciety.org"
        },
        "homepage": "https://elifesciences.org/",
        "id": "https://elifesciences.org/",
        "logo": "https://sciety.org/static/groups/elife--b560187e-f2fb-4ff9-a861-a204f3fc0fb0.png",
        "name": "eLife"
    },
    "first-step": "_:b0",
    "steps": {
        "_:b0": {
            "actions": [
                {
                    "participants": [],
                    "outputs": [
                        {
                            "type": "preprint",
                            "identifier": "100788",
                            "doi": "10.7554/eLife.100788.1",
                            "versionIdentifier": "1",
                            "license": "http://creativecommons.org/licenses/by/4.0/"
                        }
                    ]
                }
            ],
            "assertions": [
                {
                    "item": {
                        "type": "preprint",
                        "doi": "10.1101/2024.06.18.599531",
                        "versionIdentifier": "1"
                    },
                    "status": "under-review",
                    "happened": "2024-06-26T18:16:44+00:00"
                },
                {
                    "item": {
                        "type": "preprint",
                        "doi": "10.7554/eLife.100788.1",
                        "versionIdentifier": "1"
                    },
                    "status": "draft"
                }
            ],
            "inputs": [
                {
                    "type": "preprint",
                    "doi": "10.1101/2024.06.18.599531",
                    "url": "https://www.biorxiv.org/content/10.1101/2024.06.18.599531v1",
                    "versionIdentifier": "1",
                    "published": "2024-06-19",
                    "content": [
                        {
                            "type": "computer-file",
                            "url": "s3://transfers-elife/biorxiv_Current_Content/June_2024/21_Jun_24_Batch_2015/f44d05d3-6c53-1014-b4c9-de258731278c.meca"
                        }
                    ]
                }
            ],
            "next-step": "_:b1"
        },
        "_:b1": {
            "actions": [
                {
                    "participants": [],
                    "outputs": [
                        {
                            "type": "reply",
                            "published": "2024-11-11T13:14:06.397133+00:00",
                            "doi": "10.7554/eLife.100788.1.sa0",
                            "license": "http://creativecommons.org/licenses/by/4.0/",
                            "url": "https://doi.org/10.7554/eLife.100788.1.sa0",
                            "content": [
                                {
                                    "type": "web-page",
                                    "url": "https://hypothes.is/a/1Gd_FqAuEe-4Qt-BfZHckA"
                                },
                                {
                                    "type": "web-page",
                                    "url": "https://sciety.org/articles/activity/10.1101/2024.06.18.599531#hypothesis:1Gd_FqAuEe-4Qt-BfZHckA"
                                },
                                {
                                    "type": "web-page",
                                    "url": "https://sciety.org/evaluations/hypothesis:1Gd_FqAuEe-4Qt-BfZHckA/content"
                                }
                            ]
                        }
                    ]
                },
                {
                    "participants": [
                        {
                            "actor": {
                                "name": "anonymous",
                                "type": "person"
                            },
                            "role": "peer-reviewer"
                        }
                    ],
                    "outputs": [
                        {
                            "type": "review-article",
                            "published": "2024-11-11T13:14:06.722996+00:00",
                            "doi": "10.7554/eLife.100788.1.sa1",
                            "license": "http://creativecommons.org/licenses/by/4.0/",
                            "url": "https://doi.org/10.7554/eLife.100788.1.sa1",
                            "content": [
                                {
                                    "type": "web-page",
                                    "url": "https://hypothes.is/a/1JcU2KAuEe-IXx-RbKJRBg"
                                },
                                {
                                    "type": "web-page",
                                    "url": "https://sciety.org/articles/activity/10.1101/2024.06.18.599531#hypothesis:1JcU2KAuEe-IXx-RbKJRBg"
                                },
                                {
                                    "type": "web-page",
                                    "url": "https://sciety.org/evaluations/hypothesis:1JcU2KAuEe-IXx-RbKJRBg/content"
                                }
                            ]
                        }
                    ]
                },
                {
                    "participants": [
                        {
                            "actor": {
                                "name": "anonymous",
                                "type": "person"
                            },
                            "role": "peer-reviewer"
                        }
                    ],
                    "outputs": [
                        {
                            "type": "review-article",
                            "published": "2024-11-11T13:14:07.540804+00:00",
                            "doi": "10.7554/eLife.100788.1.sa2",
                            "license": "http://creativecommons.org/licenses/by/4.0/",
                            "url": "https://doi.org/10.7554/eLife.100788.1.sa2",
                            "content": [
                                {
                                    "type": "web-page",
                                    "url": "https://hypothes.is/a/1RN3WKAuEe-jvIe9ARUAkg"
                                },
                                {
                                    "type": "web-page",
                                    "url": "https://sciety.org/articles/activity/10.1101/2024.06.18.599531#hypothesis:1RN3WKAuEe-jvIe9ARUAkg"
                                },
                                {
                                    "type": "web-page",
                                    "url": "https://sciety.org/evaluations/hypothesis:1RN3WKAuEe-jvIe9ARUAkg/content"
                                }
                            ]
                        }
                    ]
                },
                {
                    "participants": [
                        {
                            "actor": {
                                "name": "anonymous",
                                "type": "person"
                            },
                            "role": "peer-reviewer"
                        }
                    ],
                    "outputs": [
                        {
                            "type": "review-article",
                            "published": "2024-11-11T13:14:08.338039+00:00",
                            "doi": "10.7554/eLife.100788.1.sa3",
                            "license": "http://creativecommons.org/licenses/by/4.0/",
                            "url": "https://doi.org/10.7554/eLife.100788.1.sa3",
                            "content": [
                                {
                                    "type": "web-page",
                                    "url": "https://hypothes.is/a/1Y1zPKAuEe-xyUMQqViEtQ"
                                },
                                {
                                    "type": "web-page",
                                    "url": "https://sciety.org/articles/activity/10.1101/2024.06.18.599531#hypothesis:1Y1zPKAuEe-xyUMQqViEtQ"
                                },
                                {
                                    "type": "web-page",
                                    "url": "https://sciety.org/evaluations/hypothesis:1Y1zPKAuEe-xyUMQqViEtQ/content"
                                }
                            ]
                        }
                    ]
                },
                {
                    "participants": [
                        {
                            "actor": {
                                "type": "person",
                                "name": "Vincent Lynch",
                                "firstName": "Vincent",
                                "_middleName": "J",
                                "surname": "Lynch",
                                "_relatesToOrganization": "University at Buffalo, State University of New York, United States of America",
                                "affiliation": {
                                    "type": "organization",
                                    "name": "University at Buffalo, State University of New York",
                                    "location": "Buffalo, United States of America"
                                }
                            },
                            "role": "editor"
                        },
                        {
                            "actor": {
                                "type": "person",
                                "name": "George Perry",
                                "firstName": "George",
                                "_middleName": "H",
                                "surname": "Perry",
                                "_relatesToOrganization": "Pennsylvania State University, United States of America",
                                "affiliation": {
                                    "type": "organization",
                                    "name": "Pennsylvania State University",
                                    "location": "University Park, United States of America"
                                }
                            },
                            "role": "senior-editor"
                        }
                    ],
                    "outputs": [
                        {
                            "type": "evaluation-summary",
                            "published": "2024-11-11T13:14:09.095356+00:00",
                            "doi": "10.7554/eLife.100788.1.sa4",
                            "license": "http://creativecommons.org/licenses/by/4.0/",
                            "url": "https://doi.org/10.7554/eLife.100788.1.sa4",
                            "content": [
                                {
                                    "type": "web-page",
                                    "url": "https://hypothes.is/a/1gE03qAuEe-OagvtuVp4VQ"
                                },
                                {
                                    "type": "web-page",
                                    "url": "https://sciety.org/articles/activity/10.1101/2024.06.18.599531#hypothesis:1gE03qAuEe-OagvtuVp4VQ"
                                },
                                {
                                    "type": "web-page",
                                    "url": "https://sciety.org/evaluations/hypothesis:1gE03qAuEe-OagvtuVp4VQ/content"
                                }
                            ]
                        }
                    ]
                },
                {
                    "participants": [],
                    "outputs": [
                        {
                            "type": "preprint",
                            "identifier": "100788",
                            "doi": "10.7554/eLife.100788.1",
                            "versionIdentifier": "1",
                            "license": "http://creativecommons.org/licenses/by/4.0/",
                            "content": [
                                {
                                    "type": "computer-file",
                                    "url": "s3://prod-elife-epp-meca/reviewed-preprints/100788-v1-meca.zip"
                                }
                            ]
                        }
                    ]
                }
            ],
            "assertions": [
                {
                    "item": {
                        "type": "preprint",
                        "doi": "10.1101/2024.06.18.599531",
                        "versionIdentifier": "1"
                    },
                    "status": "peer-reviewed"
                }
            ],
            "inputs": [
                {
                    "type": "preprint",
                    "doi": "10.1101/2024.06.18.599531",
                    "url": "https://www.biorxiv.org/content/10.1101/2024.06.18.599531v1",
                    "versionIdentifier": "1"
                }
            ],
            "next-step": "_:b2",
            "previous-step": "_:b0"
        },
        "_:b2": {
            "actions": [
                {
                    "participants": [],
                    "outputs": [
                        {
                            "type": "preprint",
                            "identifier": "100788",
                            "doi": "10.7554/eLife.100788.1",
                            "versionIdentifier": "1",
                            "license": "http://creativecommons.org/licenses/by/4.0/",
                            "published": "2024-11-19T11:00:00+00:00",
                            "partOf": {
                                "type": "manuscript",
                                "doi": "10.7554/eLife.100788",
                                "identifier": "100788",
                                "subjectDisciplines": [
                                    "Evolutionary Biology"
                                ],
                                "published": "2024-11-19T11:00:00+00:00",
                                "volumeIdentifier": "13",
                                "electronicArticleIdentifier": "RP100788",
                                "complement": []
                            }
                        }
                    ]
                }
            ],
            "assertions": [
                {
                    "item": {
                        "type": "preprint",
                        "doi": "10.7554/eLife.100788.1",
                        "versionIdentifier": "1"
                    },
                    "status": "manuscript-published"
                }
            ],
            "inputs": [
                {
                    "type": "preprint",
                    "doi": "10.1101/2024.06.18.599531",
                    "url": "https://www.biorxiv.org/content/10.1101/2024.06.18.599531v1",
                    "versionIdentifier": "1"
                },
                {
                    "type": "reply",
                    "doi": "10.7554/eLife.100788.1.sa0"
                },
                {
                    "type": "review-article",
                    "doi": "10.7554/eLife.100788.1.sa1"
                },
                {
                    "type": "review-article",
                    "doi": "10.7554/eLife.100788.1.sa2"
                },
                {
                    "type": "review-article",
                    "doi": "10.7554/eLife.100788.1.sa3"
                },
                {
                    "type": "evaluation-summary",
                    "doi": "10.7554/eLife.100788.1.sa4"
                }
            ],
            "next-step": "_:b3",
            "previous-step": "_:b1"
        },
        "_:b3": {
            "actions": [
                {
                    "participants": [],
                    "outputs": [
                        {
                            "type": "preprint",
                            "identifier": "100788",
                            "doi": "10.7554/eLife.100788.1",
                            "versionIdentifier": "1",
                            "license": "http://creativecommons.org/licenses/by/4.0/"
                        }
                    ]
                }
            ],
            "assertions": [
                {
                    "item": {
                        "type": "preprint",
                        "doi": "10.1101/2024.06.18.599531",
                        "versionIdentifier": "1"
                    },
                    "status": "under-review",
                    "happened": "2024-06-26T18:16:44+00:00"
                },
                {
                    "item": {
                        "type": "preprint",
                        "doi": "10.7554/eLife.100788.1",
                        "versionIdentifier": "1"
                    },
                    "status": "draft"
                }
            ],
            "inputs": [
                {
                    "type": "preprint",
                    "doi": "10.1101/2024.06.18.599531",
                    "url": "https://www.biorxiv.org/content/10.1101/2024.06.18.599531v1",
                    "versionIdentifier": "1",
                    "published": "2024-06-19",
                    "content": [
                        {
                            "type": "computer-file",
                            "url": "s3://transfers-elife/biorxiv_Current_Content/June_2024/21_Jun_24_Batch_2015/f44d05d3-6c53-1014-b4c9-de258731278c.meca"
                        }
                    ]
                }
            ],
            "next-step": "_:b4",
            "previous-step": "_:b2"
        },
        "_:b4": {
            "actions": [
                {
                    "participants": [],
                    "outputs": [
                        {
                            "type": "reply",
                            "published": "2024-11-11T13:14:06.397133+00:00",
                            "doi": "10.7554/eLife.100788.1.sa0",
                            "license": "http://creativecommons.org/licenses/by/4.0/",
                            "url": "https://doi.org/10.7554/eLife.100788.1.sa0",
                            "content": [
                                {
                                    "type": "web-page",
                                    "url": "https://hypothes.is/a/1Gd_FqAuEe-4Qt-BfZHckA"
                                },
                                {
                                    "type": "web-page",
                                    "url": "https://sciety.org/articles/activity/10.1101/2024.06.18.599531#hypothesis:1Gd_FqAuEe-4Qt-BfZHckA"
                                },
                                {
                                    "type": "web-page",
                                    "url": "https://sciety.org/evaluations/hypothesis:1Gd_FqAuEe-4Qt-BfZHckA/content"
                                }
                            ]
                        }
                    ]
                },
                {
                    "participants": [
                        {
                            "actor": {
                                "name": "anonymous",
                                "type": "person"
                            },
                            "role": "peer-reviewer"
                        }
                    ],
                    "outputs": [
                        {
                            "type": "review-article",
                            "published": "2024-11-11T13:14:06.722996+00:00",
                            "doi": "10.7554/eLife.100788.1.sa1",
                            "license": "http://creativecommons.org/licenses/by/4.0/",
                            "url": "https://doi.org/10.7554/eLife.100788.1.sa1",
                            "content": [
                                {
                                    "type": "web-page",
                                    "url": "https://hypothes.is/a/1JcU2KAuEe-IXx-RbKJRBg"
                                },
                                {
                                    "type": "web-page",
                                    "url": "https://sciety.org/articles/activity/10.1101/2024.06.18.599531#hypothesis:1JcU2KAuEe-IXx-RbKJRBg"
                                },
                                {
                                    "type": "web-page",
                                    "url": "https://sciety.org/evaluations/hypothesis:1JcU2KAuEe-IXx-RbKJRBg/content"
                                }
                            ]
                        }
                    ]
                },
                {
                    "participants": [
                        {
                            "actor": {
                                "name": "anonymous",
                                "type": "person"
                            },
                            "role": "peer-reviewer"
                        }
                    ],
                    "outputs": [
                        {
                            "type": "review-article",
                            "published": "2024-11-11T13:14:07.540804+00:00",
                            "doi": "10.7554/eLife.100788.1.sa2",
                            "license": "http://creativecommons.org/licenses/by/4.0/",
                            "url": "https://doi.org/10.7554/eLife.100788.1.sa2",
                            "content": [
                                {
                                    "type": "web-page",
                                    "url": "https://hypothes.is/a/1RN3WKAuEe-jvIe9ARUAkg"
                                },
                                {
                                    "type": "web-page",
                                    "url": "https://sciety.org/articles/activity/10.1101/2024.06.18.599531#hypothesis:1RN3WKAuEe-jvIe9ARUAkg"
                                },
                                {
                                    "type": "web-page",
                                    "url": "https://sciety.org/evaluations/hypothesis:1RN3WKAuEe-jvIe9ARUAkg/content"
                                }
                            ]
                        }
                    ]
                },
                {
                    "participants": [
                        {
                            "actor": {
                                "name": "anonymous",
                                "type": "person"
                            },
                            "role": "peer-reviewer"
                        }
                    ],
                    "outputs": [
                        {
                            "type": "review-article",
                            "published": "2024-11-11T13:14:08.338039+00:00",
                            "doi": "10.7554/eLife.100788.1.sa3",
                            "license": "http://creativecommons.org/licenses/by/4.0/",
                            "url": "https://doi.org/10.7554/eLife.100788.1.sa3",
                            "content": [
                                {
                                    "type": "web-page",
                                    "url": "https://hypothes.is/a/1Y1zPKAuEe-xyUMQqViEtQ"
                                },
                                {
                                    "type": "web-page",
                                    "url": "https://sciety.org/articles/activity/10.1101/2024.06.18.599531#hypothesis:1Y1zPKAuEe-xyUMQqViEtQ"
                                },
                                {
                                    "type": "web-page",
                                    "url": "https://sciety.org/evaluations/hypothesis:1Y1zPKAuEe-xyUMQqViEtQ/content"
                                }
                            ]
                        }
                    ]
                },
                {
                    "participants": [
                        {
                            "actor": {
                                "type": "person",
                                "name": "Vincent Lynch",
                                "firstName": "Vincent",
                                "_middleName": "J",
                                "surname": "Lynch",
                                "_relatesToOrganization": "University at Buffalo, State University of New York, United States of America",
                                "affiliation": {
                                    "type": "organization",
                                    "name": "University at Buffalo, State University of New York",
                                    "location": "Buffalo, United States of America"
                                }
                            },
                            "role": "editor"
                        },
                        {
                            "actor": {
                                "type": "person",
                                "name": "George Perry",
                                "firstName": "George",
                                "_middleName": "H",
                                "surname": "Perry",
                                "_relatesToOrganization": "Pennsylvania State University, United States of America",
                                "affiliation": {
                                    "type": "organization",
                                    "name": "Pennsylvania State University",
                                    "location": "University Park, United States of America"
                                }
                            },
                            "role": "senior-editor"
                        }
                    ],
                    "outputs": [
                        {
                            "type": "evaluation-summary",
                            "published": "2024-11-11T13:14:09.095356+00:00",
                            "doi": "10.7554/eLife.100788.1.sa4",
                            "license": "http://creativecommons.org/licenses/by/4.0/",
                            "url": "https://doi.org/10.7554/eLife.100788.1.sa4",
                            "content": [
                                {
                                    "type": "web-page",
                                    "url": "https://hypothes.is/a/1gE03qAuEe-OagvtuVp4VQ"
                                },
                                {
                                    "type": "web-page",
                                    "url": "https://sciety.org/articles/activity/10.1101/2024.06.18.599531#hypothesis:1gE03qAuEe-OagvtuVp4VQ"
                                },
                                {
                                    "type": "web-page",
                                    "url": "https://sciety.org/evaluations/hypothesis:1gE03qAuEe-OagvtuVp4VQ/content"
                                }
                            ]
                        }
                    ]
                },
                {
                    "participants": [],
                    "outputs": [
                        {
                            "type": "preprint",
                            "identifier": "100788",
                            "doi": "10.7554/eLife.100788.1",
                            "versionIdentifier": "1",
                            "license": "http://creativecommons.org/licenses/by/4.0/",
                            "content": [
                                {
                                    "type": "computer-file",
                                    "url": "s3://prod-elife-epp-meca/reviewed-preprints/100788-v1-meca.zip"
                                }
                            ]
                        }
                    ]
                }
            ],
            "assertions": [
                {
                    "item": {
                        "type": "preprint",
                        "doi": "10.1101/2024.06.18.599531",
                        "versionIdentifier": "1"
                    },
                    "status": "peer-reviewed"
                }
            ],
            "inputs": [
                {
                    "type": "preprint",
                    "doi": "10.1101/2024.06.18.599531",
                    "url": "https://www.biorxiv.org/content/10.1101/2024.06.18.599531v1",
                    "versionIdentifier": "1"
                }
            ],
            "next-step": "_:b5",
            "previous-step": "_:b3"
        },
        "_:b5": {
            "actions": [
                {
                    "participants": [],
                    "outputs": [
                        {
                            "type": "preprint",
                            "identifier": "100788",
                            "doi": "10.7554/eLife.100788.1",
                            "versionIdentifier": "1",
                            "license": "http://creativecommons.org/licenses/by/4.0/",
                            "published": "2024-11-19T11:00:00+00:00",
                            "partOf": {
                                "type": "manuscript",
                                "doi": "10.7554/eLife.100788",
                                "identifier": "100788",
                                "subjectDisciplines": [
                                    "Evolutionary Biology"
                                ],
                                "published": "2024-11-19T11:00:00+00:00",
                                "volumeIdentifier": "13",
                                "electronicArticleIdentifier": "RP100788",
                                "complement": []
                            }
                        }
                    ]
                }
            ],
            "assertions": [
                {
                    "item": {
                        "type": "preprint",
                        "doi": "10.7554/eLife.100788.1",
                        "versionIdentifier": "1"
                    },
                    "status": "manuscript-published"
                }
            ],
            "inputs": [
                {
                    "type": "preprint",
                    "doi": "10.1101/2024.06.18.599531",
                    "url": "https://www.biorxiv.org/content/10.1101/2024.06.18.599531v1",
                    "versionIdentifier": "1"
                },
                {
                    "type": "reply",
                    "doi": "10.7554/eLife.100788.1.sa0"
                },
                {
                    "type": "review-article",
                    "doi": "10.7554/eLife.100788.1.sa1"
                },
                {
                    "type": "review-article",
                    "doi": "10.7554/eLife.100788.1.sa2"
                },
                {
                    "type": "review-article",
                    "doi": "10.7554/eLife.100788.1.sa3"
                },
                {
                    "type": "evaluation-summary",
                    "doi": "10.7554/eLife.100788.1.sa4"
                }
            ],
            "previous-step": "_:b4"
        }
    }
}
```

**Comment by fred-atherden** [comment]
Interesting - thanks Graham!
